src/apps/Permissions/ManageRolePanel.js

```diff
@@ -286,16 +286,18 @@ class ManageRolePanel extends React.PureComponent {
             margin-top: ${3 * GU}px;
           `}
         >
-          <Field label="App">
+          <Field label="On app">
             {app && (
               <AppInstanceLabel app={app} proxyAddress={app.proxyAddress} />
             )}
           </Field>
 
-          <Field label="Action description">{role && role.name}</Field>
+          <Field label="Permission">
```

Comment:
@dizzypaty I've changed this to be "Permission" instead of "Action" since "Action" is repeated below

Reply:
I think this is ok and we can revisit this terminology after the launch and some user validation. I feel strongly about calling it 'Action' rather than 'Permission' because I understand the permission as the composition of 4 elements: 'On app' + 'Action' + 'Assigned to entity' + 'Managed by' = Permission.

Also, for consistency purposes, do you reckon that we should change 'Action' to 'Permission' on the available permissions table to match the title on the side panel?